WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -222,11 +222,16 @@ - (BlogDetailsSection *)generalSectionViewModel
                                                  }]];
 
     if ([Feature enabled:FeatureFlagPlans] && [self.blog supports:BlogFeaturePlans]) {
-        [rows addObject:[[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Plans", @"Action title. Noun. Links to a blog's Plans screen.")
-                                                        image:[UIImage imageNamed:@"icon-menu-plans"]
-                                                     callback:^{
-                                                         [weakSelf showPlans];
-                                                     }]];
+        BlogDetailsRow *row = [[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Plans", @"Action title. Noun. Links to a blog's Plans screen.")
+                                                              image:[UIImage imageNamed:@"icon-menu-plans"]
+                                                           callback:^{
+                                                               [weakSelf showPlans];
+                                                           }];
+
+        Plan *plan = [[Plan alloc] initWithPlan:[self.blog.planID integerValue]];
```

Comment:
Thanks for catching this! I've updated `PlanObjc` to use a failable initializer.